cmd/event-received/sirius_event_handler.go

```diff
@@ -477,6 +477,7 @@ func handleCertificateProviderIdentityCheckedFailed(ctx context.Context, dynamoC
 	localizer := bundle.For(provided.Donor.ContactLanguagePreference)
 
 	return notifyClient.SendActorEmail(ctx, notify.ToDonor(provided), v.UID, notify.InformDonorPaperCertificateProviderIdentityCheckFailed{
+		Greeting:                    notifyClient.EmailGreetingProvided(provided),
```

Comment:
yeah, it is worth reverting and using the lpa-store data. Could probably stop using `getDonorByLpaUID` and use the lpa for everything here?